functionals/botpress-nlu/src/providers/dialogflow.js

```diff
@@ -75,6 +75,7 @@ export default class DialogflowProvider extends Provider {
     return {
       intent,
       intents: [intent],
+      original: detection,
```

Comment:
I used `origintal` as the key to store the original response in, but feel free to adjust this.